terraform/variables.tf

```diff
@@ -115,9 +110,14 @@ variable "loader_sentry_dsn" {
   sensitive   = true
 }
 
-variable "bastion_security_group_id" {
-  description = "ID for a security group in AWS that allows access to a bastion server inside our VPC. SSHing into this server will allow access to any services that allow this security group."
-  default     = "sg-06fc404762de692db"
+variable "bastion_key_name" {
+  description = "The key pair name to use for running the bastion server"
+  default     = "univaf-bastion-key"
+}
+
+variable "bastion_ami_group_id" {
+  description = "AMI to be used for the bastion server"
+  default = "ami-03d5c68bab01f3496"
```

Comment:
Are these in preparation for a bastion config that does not actually exist on this PR yet? I don’t see them actually referenced anywhere.

Reply:
ah yeah this was when i was attempting to import the ec2 instance in to this, i should remove it.